Frameworks/include/CGIWICBitmap.h

```diff
@@ -34,77 +35,84 @@ class CGIWICBitmap;
 
 class CGIWICBitmapLock : public RuntimeClass<RuntimeClassFlags<RuntimeClassType::ClassicCom>, IAgileObject, FtmBase, IWICBitmapLock> {
 public:
-    CGIWICBitmapLock(_In_ CGImageBacking* imageBacking, _In_ WICPixelFormatGUID pixelFormat)
-        : m_imageBacking(imageBacking), m_pixelFormat(pixelFormat) {
-        m_dataBuffer = static_cast<BYTE*>(m_imageBacking->LockImageData());
+    CGIWICBitmapLock(_In_ IDisplayTexture* texture, _In_ const WICRect* region, _In_ WICPixelFormatGUID pixelFormat)
+        : m_texture(texture), m_pixelFormat(pixelFormat), m_locked_rect(region) {
+        int bpr;
+        m_dataBuffer = static_cast<BYTE*>(m_texture->Lock(&bpr));
+        m_bytesPerRow = static_cast<size_t>(bpr);
```

Comment:
With in CG it's mostly bpr. 
I'll leave it as is for now.

---
In reply to: [86919975](https://github.com/Microsoft/WinObjC/pull/1332#discussion_r86919975) [](ancestors = 86919975)